madminer/utils/interfaces/mg.py

```diff
@@ -536,9 +517,9 @@ def setup_mg_reweighting_with_scripts(
     make_file_executable(script_file)
 
     # How to call it from master script
-    call_instruction = "{}/{} [MG_process_directory] [log_directory]".format(
-        mg_process_directory, script_file_from_mgprocdir
-    )
+    call_instruction = \
```

Comment:
See https://github.com/diana-hep/madminer/pull/438/files#r512226711

Reply:
Ok. Changing it to `( ... )` style.